test/dynamo/test_logging.py

```diff
@@ -763,10 +769,11 @@ def fn(x):
         self.assertGreater(len(records), 0)
         self.assertLess(len(records), 4)
 
+    @xfailIfXPU  # https://github.com/pytorch/pytorch/issues/157778
```

Comment:
Could we skip this case on XPU and link the issue, instead of adding a new decorator `xfailIfXPU`?

Reply:
You are right, let me do the update.